src/components/Navbar/navbar.tsx

```diff
@@ -0,0 +1,25 @@
+export default function Navbar() {
```

Comment:
Let's use the arrow syntax as this is more modern practice
`export const Navbar = () => {...}`